content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md

```diff
@@ -45,6 +45,10 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
  ``` shell
 $ ghe-dep-graph-enable
 ```
+{% note %}
+**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."
+{% endnote %}
```

Comment:
The note needs an empty line after the opening tag and before the closing tag. Otherwise this change looks good, thanks.

```suggestion
{% note %}

**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."

{% endnote %}
```